### DISCUSSION TITLE
Ability to choose different skins for different rulesets

### DISCUSSION THREAD

**Discussion by theredlumas2** [ppy/osu#22727] [Ideas] [answered]
For example, you can choose the classic osu! skin for regular and argon for mania. then, when loading up said rulesets, it will use the skins you selected.

**Comment by mk56-spn** [comment]
This is already planned. Iirc frenzibyte was working on it until recently. I think ppy might've taken over with his recent skinning overhaul. In any case it will come

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/22388

**Comment by AbdeI1** [comment]
I also think this would be a good idea. For me, I often switch between rulesets while playing and really dislike some off the mania skins with the skins that I use, so I also switch skins. While #22388 adds additional customization options for skins that allows layouts to be changed per ruleset, it still only allows one skin to be used globally. 

This would mean that if I really liked one skin's version of the classic ruleset, but another skin's version of the mania ruleset (which is often the case), I would not be able easily switch between them unless I choose to modify one of the skins. Even then, the existing modification system only allows changing the layout, but there doesn't seem to be a way to mix and match assets from one skin to another within the the game. I can't figure out a way to change the keys being used in mania for example.

An extra option that simply changes the current skin whenever the ruleset is changed is not too complex, in my opinion, and would solve this minor inconvenience that I have where I always manually change skins whenever I switch rulesets.
